THANKS.txt

```diff
@@ -199,6 +199,7 @@ Kai Striega for improvements to the scipy.optimize.linprog simplex method.
 Josua Sassen for improvements to scipy.interpolate.Rbf
 Stiaan Gerber for a bug fix in scipy.optimize.
 Nicolas Hug for the Yeo-Johnson transformation.
+Idan David for the log survival function and log cummulative distribution function of the hypergeometric distribution
```

Comment:
typo "cummulative" and these are actually modifications to those functions, right?

Reply:
Cleaned up a bit in #10091